packages/dcos-integration-test/extra/test_mesos.py

```diff
@@ -76,9 +60,21 @@ def test_if_marathon_app_can_be_debugged(dcos_api_session):
             if agent['id'] == agent_id:
                 agent_hostname = agent['hostname']
         assert agent_hostname is not None, 'Agent hostname not found for agent_id {}'.format(agent_id)
-        agent_v1_url = 'http://{}:{}/api/v1'.format(agent_hostname, 5051)
         logging.debug('Located %s with containerID %s on agent %s', app_id, container_id, agent_hostname)
 
+        # Wrapper to post, log, and validate return status
```

Comment:
I dropped the comment - it's a two-line function that is clearer than the comment itself.